test/e2e/servicebindingrequest_test.go

```diff
@@ -29,10 +30,12 @@ import (
 type Step string
 
 const (
-	DBStep  Step = "create-db"
-	AppStep Step = "create-app"
-	SBRStep Step = "create-sbr"
-	CSVStep Step = "create-csv"
+	DBStep               Step = "create-db"
+	AppStep              Step = "create-app"
+	SBRStep              Step = "create-sbr"
+	CSVStep              Step = "create-csv"
+	deploymentsGVR            = "deployments"
```

Comment:
Wouldn't the following be slightly more readable?

```suggestion
	deploymentsGVR            = schema.GroupVersionResource{Group: "apps", Version: "v1", Resource: "deployments"}
```

Reply:
Done eb3f22b